bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/internal/models/v2/form/FormContainerImpl.java

```diff
@@ -196,7 +196,12 @@ public String getId() {
     @JsonIgnore
     @Nullable
     public String getRedirectUrl() {
-        return getContextPath() + GuideUtils.getRedirectUrl(redirect, getPath());
+        String redirectURL = redirect;
```

Comment:
Please add test case for this change here, https://github.com/adobe/aem-core-forms-components/blob/master/ui.tests/test-module/specs/actions/submit/submit.runtime.spec.js#L102-L122

Reply:
added test, but waiting for page load times out.